neuro_data/static_images/data_schemas.py

```diff
@@ -522,6 +559,10 @@ def compute_data(self, key):
         if len(images.shape) == 3:
             log.info('Adding channel dimension')
             images = images[:, None, ...]
+        elif len(images.shape) == 4:
+            print('hihihi')
+            images = images.transpose(0, 3, 1, 2)
+        print(images.shape)
```

Comment:
if you still wanna print this shape, maybe modify to say `print('Image shapes', images.shape)`

Reply:
Also debugging statements, I ran into a problem there and was trying to figure out how to transpose it correctly